src/index.ts

```diff
@@ -125,6 +125,10 @@ function normalizeArray<T>(obj?: T[]): T[] | undefined {
 }
 
 function onError(err: AxiosError) {
+  if (axios.isCancel(err)) {
```

Comment:
Thanks for the fix!  Could I trouble you with a test to validate the behavior?  Otherwise I could easily see myself breaking this again 😆

Reply:
I've added a new test and changed the return to Promise.reject(err).